content/300-guides/150-testing/100-unit-testing.mdx

```diff
@@ -69,37 +69,36 @@ npm install jest-mock-extended --save-dev
 
 ### Singleton
 
-
 The following steps guide you through mocking the Prisma Client using a singleton pattern.
 
 1. Create a file at your projects root called `client.ts` and add the following code. This will instantiate a Prisma client instance.
 
-  ```ts file=client.ts
-  import { PrismaClient } from '@prisma/client'
+```ts file=client.ts
+import { PrismaClient } from '@prisma/client'
 
-  const prisma = new PrismaClient()
-  export default prisma
-  ```
+const prisma = new PrismaClient()
+export default prisma
+```
```

Comment:
These code blocks should be under the list item - so probably 3 spaces needed to render correctly.

Reply:
Looks correct

<img width="773" alt="image" src="https://user-images.githubusercontent.com/4734297/124765891-daba8780-df4f-11eb-97d5-5848070dfee3.png">